### DISCUSSION TITLE
Model Not Showing Up In Model-Viewer

### DISCUSSION THREAD

**Discussion by jealouscase** [google/model-viewer#1833] [Q&A] [answered]
Hello, I'm having some trouble with Google Model-Viewer in Glitch.com. I have an index.html, script.js, style.css file, with an assets folder. My index.html code is below:
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>&lt;model-viewer&gt; example</title>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <link rel="stylesheet" href="style.css">
    
    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
    
    <style>
      model-viewer {
        width: 400px;
        height: 600px;
        margin: 0 auto;
      }
    </style>

  </head> 
  
  <body>
    <model-viewer id="reveal" loading="eager" camera-controls auto-rotate src="https://cdn.glitch.com/db911cdb-a190-462a-a6ae-fd9c5aa7e892%2FLightBulb_01.gltf?v=1610761837542"></model-viewer>
  </body>
</html>
```
The src for the model comes from my assets folder, where I uploaded, and tried, both the original and updated versions of the glTF file from this Poly.Google page: https://cdn.glitch.com/db911cdb-a190-462a-a6ae-fd9c5aa7e892%2FLightBulb_01.gltf?v=1610768352087

When I open my page, I can tell there is a model viewer window (by the loading bar) at the width, height, and margin I specified, but the model does not show up. Any suggestions would be appreciated :D

**Comment by nshutijean** [comment, marked as answer]
If you are using Brave browser or any browser that blocks fingerprinting for security purposes, then the issue will persist. Again the link to your model returns a 404. So i used a model locally, allowed fingerprinting in the browser and the code works fine.

browser issue: https://github.com/mrdoob/three.js/issues/16904

screenshot:
![brave-model-viewer](https://user-images.githubusercontent.com/28356145/104838658-2788a400-58c5-11eb-9ff2-725023a6b5ba.png)
